Firestore/core/src/model/value_util.h

```diff
@@ -80,10 +80,20 @@ bool Equals(const google_firestore_v1_ArrayValue& left,
  */
 std::string CanonicalId(const google_firestore_v1_Value& value);
 
-/** Returns the lowest value for the given value type (inclusive). */
+/**
+ * Returns the lowest value for the given value type (inclusive).
+ *
+ * The returned value might point to global variables, and the caller is not
+ * supposed to own or release it, without calling `DeepClone`.
```

Comment:
Maybe: The returned value might point to heap allocated memory that is owned by this function. To take ownership of this memory, invoke `DeepClone`.

Reply:
Done.